numpy/core/src/umath/ufunc_object.c

```diff
@@ -5915,7 +5915,12 @@ trivial_at_loop(PyArrayMethodObject *ufuncimpl, NPY_ARRAYMETHOD_FLAGS flags,
         steps[2] = 0;
     } else {
         args[2] = (char *)PyArray_DATA(op2_array);
-        steps[2] = PyArray_STRIDE(op2_array, 0);
+        if (PyArray_NDIM(op2_array) == 0
```

Comment:
This was incorrectly `PyArray_DIMS` -- maybe the failure was related. Anyway, now fixed.